mitodl/open-discussions
# Remove INDEX_UPDATES feature flag

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2521 

#### What's this PR do?
Removes the `INDEX_UPDATES` feature flag so functions which were previously conditional on it are enabled permanently

#### How should this be manually tested?
Update your profile with some word which is unique, then go to `/search` and find your profile there. If the word shows up, the indexing should still be functioning
